### PR TITLE
Fixed bug behavior

### PR DESCRIPTION
**Pull request by Tequelequeteque**
Behavior equal create migration

### CODE REVIEW

**Comment by imnotjames** [comment]
Can you clarify what the bug is?

File: src/commands/EntityCreateCommand.ts [hunk 1]
```diff
@@ -51,13 +51,9 @@ export class EntityCreateCommand implements yargs.CommandModule {
                 } catch (err) { }
             }
 
-            const path = process.cwd() + "/" + (directory ? (directory + "/") : "") + filename;
-            const fileExists = await CommandUtils.fileExists(path);
-            if (fileExists) {
-                throw `File ${chalk.blue(path)} already exists`;
-            }
+              const path = (directory.startsWith("/") ? "" : process.cwd() + "/") + (directory ? (directory + "/") : "") + filename;
```
Comment (imnotjames) [inline src/commands/EntityCreateCommand.ts:54]: It kinda looks to me that you're trying to handle absolute paths.

In that case, perhaps look at `path.resolve`?  Something like `path.resolve(directory, filename)` would get you probably much of the behavior to handle both relative and absolute paths.

File: src/commands/EntityCreateCommand.ts [hunk 2]
```diff
@@ -51,13 +51,9 @@ export class EntityCreateCommand implements yargs.CommandModule {
...
             await CommandUtils.createFile(path, fileContent);
-            console.log(chalk.green(`Entity ${chalk.blue(path)} has been created successfully.`));
+            console.log(`Migration ${chalk.blue(path)} has been generated successfully.`);
```
Comment (imnotjames) [inline src/commands/EntityCreateCommand.ts:56]: Was this change accidental?

**Comment by Tequelequeteque** [comment]
I pass absolute path on cli migrateDir and directory create ok but in entityDir he create all subdir in src ...
example :
I pass /user/myuser/dir/src/entity
so he create /user/myuser/dir/src//user/myuser/dir/src/entity

**Comment by Tequelequeteque** [comment]
```ts
import { resolve, join } from 'path';
import 'dotenv/config';
import { PostgresConnectionOptions } from 'typeorm/driver/postgres/PostgresConnectionOptions';

interface IENV{
  [key:string]: any,
}

const {
  DB_TYPE,
  DB_PORT,
  DB_USER,
  DB_PASS,
  DB_NAME,
} = process.env as unknown as IENV;

const entitiesPath = resolve(__dirname, '..', 'entities');
const migrationsPath = resolve(__dirname, 'migrations');

// behavior diferent
const entitiesCliPath = entitiesPath.split('/').slice(-2).join('/');

const dbConfig = {
  synchronize: true,
  name: 'default',
  type: DB_TYPE,
  host: 'localhost',
  port: DB_PORT,
  username: DB_USER,
  password: DB_PASS,
  database: DB_NAME,
  migrationsRun: true,
  entities: [entitiesPath.concat('/**/*ts')],
  migrations: [migrationsPath.concat('/**/*ts')],
  cli: {
    entitiesDir: entitiesCliPath,
    migrationsDir: migrationsPath,
  },
  dropSchema: true,
  logging: true,
} as PostgresConnectionOptions;

export { dbConfig };

```

File: src/commands/EntityCreateCommand.ts (hunk 1 above)
Comment (Tequelequeteque) [reply src/commands/EntityCreateCommand.ts:54]: I just copy code from MigrationCreateCommand...
Maybe delegate task create file =T

File: src/commands/EntityCreateCommand.ts (hunk 2 above)
Comment (Tequelequeteque) [reply src/commands/EntityCreateCommand.ts:56]: I just copy code from MigrationCreateCommand...
Maybe delegate task create file =T

**Comment by imnotjames** [comment]
Given that this breaks the build, I'll be closing this in favor of #6807

**Pull request closed without merging**